tap_google_ads/__init__.py

```diff
@@ -12,16 +12,30 @@
 
 REQUIRED_CONFIG_KEYS = [
     "start_date",
-    "oauth_client_id",
-    "oauth_client_secret",
-    "refresh_token",
-    "customer_ids",
-    "developer_token",
+    "login_customer_ids",
+    "developer_token"
```

Comment:
Is this field needed in required config keys?  I see it also listed in additional config keys.

Reply:
Good call. It is required for both authentication methods so I will remove it from ADDDITIONAL_CONFIG_KEYS.